src/openfl/display/Stage.hx

```diff
@@ -2611,8 +2613,9 @@ class Stage extends DisplayObjectContainer #if lime implements IModule #end
 			event.localX = local.x;
 			event.localY = local.y;
 			event.target = target;
+			event.clickCount = window.clickCount
```

Comment:
Missing semicolon.

Reply:
Ill fix that in a moment. Let's forget that happened 😅